src/gamestate/dcon_generated.txt

```diff
@@ -3555,6 +3560,11 @@ object {
 		type{ array{commodity_id}{float} }
 		tag{ save }
 	}
+	property {
+		name{ intermediate_demand }
+		type{ array{commodity_id}{float} }
+		tag{ save }
```

Comment:
I don't know your formulas, so I am asking just to double check, but do these intermediate values need to persist between days? If they are reset at the start of each day, then they can be safely omitted from the save file